docs/general/ad-filtering/create-own-filters.md

```diff
@@ -409,16 +409,21 @@ This modifier defines a rule which applies only to addresses that match the case
 
 #### **`$method`** {#method-modifier}
 
-This modifier limits the rule scope to requests that use the specified set of HTTP methods. Negated methods are allowed. The methods must be specified in all lowercase characters, but are matched case-insensitively.
+This modifier limits the rule scope to requests that use the specified set of HTTP methods. Negated methods are allowed. The methods must be specified in all lowercase characters, but are matched case-insensitively. To add multiple domains to one rule, use the `|` character as a separator.
 
 **Examples**
 
 * `||evil.com^$method=get|head` blocks only GET and HEAD requests to `evil.com`.
-* `||evil.com^$method=~post|~put` blocks any requests except POST or PUT to `evil.com`.
+* `||evil.com^$method=~post|~put` blocks any requests  to `evil.com`,except POST or PUT.
+* `@@||evil.com$method=get` unblocks only GET requests to `evil.com`.
+* `@@||evil.com$method=~post` blocks any requests to `evil.com`, except POST.
+
+Please note, that rules with mixed value restriction are considered invalid. So, for example, the rule
+`||evil.com^$method=get|~head` will be rejected.
 
 > **Compatibility with different versions of AdGuard**
 >
-> `$method` is available starting with CoreLibs v1.12.
+> Rules with the `$method` are supported by AdGuard for Windows, Mac, Android, **running CoreLibs v1.12 or later**, and AdGuard Browser extension for Chrome, Firefox, Edge. 
```

Comment:
Starting with `v4.2` or `v4.3`?

Reply:
@ameshkov v4.2, but we were considering adding TSUrlFilter version here, in a similar fashion as CoreLibs